backend/Dockerfile

```
@@ -17,10 +17,11 @@ COPY apache-ports.conf /etc/apache2/ports.conf
 RUN docker-php-ext-install pdo pdo_mysql
 
 # XDebug
-RUN pecl install xdebug-2.9.6
+RUN pecl install xdebug-3.0.2
 RUN docker-php-ext-enable xdebug
 RUN touch /var/log/xdebug.log && chmod a+rw /var/log/xdebug.log
-ENV XDEBUG_CONFIG="remote_enable=1 remote_autostart=1 remote_connect_back=1 remote_host=docker-host remote_port=9000 remote_log=/var/log/xdebug.log"
+ENV XDEBUG_MODE="develop,debug"
```

Hier wäre noch cool, wenn man die coverage anschalten könnte
```suggestion
ENV XDEBUG_MODE="develop,debug,coverage"
```